dqc_us_rules/dqc_us_0046.py

```diff
@@ -33,10 +33,6 @@ def _find_errors(val):
     calc_children = _load_config(_CONFIG_JSON_FILE)
     if not calc_children:
         return  # nothing can be checked
-    # convert children lists into sets for faster "in" function processing
-    calc_children = dict(
-        (key, set(value)) for key, value in calc_children.items()
-    )
```

Comment:
You removed this which I think is good, but you do not add a line later to make a set out of the values during checking. Was that your intention? Do you need to check against a set or is a list fine?